bigquery/CHANGELOG.md

```diff
@@ -4,6 +4,26 @@
 
 [1]: https://pypi.org/project/google-cloud-bigquery/#history
 
+## 1.23.0
+
+12-11-2019 13:31 PST
+
+### New Features
+
+- add `close()` method to client for releasing open sockets ([#9894](https://github.com/googleapis/google-cloud-python/pull/9894))
+- add support of `use_avro_logical_types` for extract jobs ([#9642](https://github.com/googleapis/google-cloud-python/pull/9642))
+- add support for hive partitioning options configuration ([#9626](https://github.com/googleapis/google-cloud-python/pull/9626))
+- add description for routine entities ([#9785](https://github.com/googleapis/google-cloud-python/pull/9785))
+
+### Documentation
+
+- update code samples to use strings for table and dataset IDs ([#9495](https://github.com/googleapis/google-cloud-python/pull/9495))
+
+### Internal / Testing Changes
+
+- run unit tests with Python 3.8 ([#9880](https://github.com/googleapis/google-cloud-python/pull/9880))
+- import `Mapping` from `collections.abc` not from `collections` ([#9826](https://github.com/googleapis/google-cloud-python/pull/9826))
```

Comment:
As per style guide, the sentences should be capitalized and concluded with a period at the end, right?

Reply:
Done in f499e7eb34395a9acf2b2e7cff5e12e2515d7077.